packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -236,10 +227,7 @@ export class SearchBoxBase extends React.Component<ISearchBoxProps, ISearchBoxSt
     ev.stopPropagation();
   };
 
-  private _onBlur = (ev: React.FocusEvent<HTMLInputElement>): void => {
-    if (this._events) {
-      this._events.off();
-    }
+  private _onBlur = (ev: FocusEvent): void => {
     this.setState({
       hasFocus: false
     });
```

Comment:
to fix the red squiggle on `this.props.onBlur(ev)`, the best you can do here (I think) is to cast the `ev` to an `any` to use the `HTMLInputAttributes` onBlur definition. It's not technically perfect, but it's also a non breaking change that doesn't really matter a ton.